src/classes/CanvasRenderingContext2D.js

```diff
@@ -1395,7 +1390,11 @@ export default class CanvasRenderingContext2D {
   }
 
   get strokeStyle() {
-    return this._strokeStyleStack[this._stackIndex];
+    const color = this._strokeStyleStack[this._stackIndex];
```

Comment:
Again, I'm suggesting to return the value from the stack here to make it faster.